tests/gui/components/svg-icon.spec.js

```diff
@@ -48,7 +48,7 @@ describe('Browser: SVGIcon', function () {
       iconContents[0] = `<!--${iconContents[0].slice(1, iconContents[0].length - 1)}-->`
       iconContents = iconContents.join('\n')
 
-      const element = $compile(`<svg-icon path="'${icon}'">Resin.io</svg-icon>`)($rootScope)
+      const element = $compile(`<svg-icon paths="['${icon}']">Resin.io</svg-icon>`)($rootScope)
```

Comment:
Hmmm, so it looks like `paths` and `contents` both now actually accept arrays of values, rather than just a single value? And it then searches through each of these arrays, and displays the first value in the array that's valid? Sounds like lots more unit tests would be useful to validate this new behaviour... ;-)

(and this behaviour should also be mentioned in the jsdoc, especially the fact that 'contents' (if present) is checked before `paths` is checked)

Reply:
I've added some tests for this now